source/fe/fe_values.cc

```diff
@@ -2892,7 +2892,7 @@ namespace internal
           = &shape_derivatives[shape_func][0];
         for (unsigned int point=0; point<n_quadrature_points; ++point)
           derivatives[point] += value *
-                                dealii::Tensor<order,spacedim,Number>(*shape_derivative_ptr++);
+                                dealii::Tensor<order,spacedim>(*shape_derivative_ptr++);
```

Comment:
`shape_derivative_ptr` is already a `const Tensor<order,spacedim> *`. Hence, there is no point in calling the copy constructor explicitly. The product must in the end be converted from `const Tensor<order,spacedim>` to `const Tensor<order,spacedim,Number>`. Doing this implicitly should be fine (especially since `Number` will often be the default type `double`).

Reply:
> shape_derivative_ptr is already a const Tensor<order,spacedim> *.

Yes, of course you're right. I'll strip out the call to the copy constructor here; this would then align with the structure of these calls in other functions.